src/bootlint.js

```diff
@@ -938,6 +938,12 @@ var LocationIndex = _location.LocationIndex;
             reporter('`.form-control` cannot be used on non-textual `<input>`s, such as those whose `type` is: `file`, `checkbox`, `radio`, `range`, `button`', formControlsOnWrongTypes);
         }
     });
+    addLinter("E045", function lintNavbarButtons($, reporter) {
+        var imgResponsiveNotOnImg = $('.img-responsive:not(img)').length;
+        if (imgResponsiveNotOnImg) {
+            reporter('`.img-responsive` should only be used on `<img>`s', this);
```

Comment:
Fixed. It was the lack of a jQuery object as the second argument of reporter which caused it.